test/unit/ElectronicResourcesSearch.test.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { expect } from 'chai'
+import { Link } from 'react-router'
+
+import ElectronicResourcesSearch from "../../src/app/components/ElectronicResourcesSearch/ElectronicResourcesSearch";
+
+describe('ElectronicResourcesSearch', () => {
+  describe('with no resources', () => {
+    it('should render null', () => {
+      const component = mount(<ElectronicResourcesSearch resources={[]}/>);
+      expect(component.html()).to.be.null
+    });
+  });
+
+  describe('with at least one resource', () => {
+    let onClickRecord;
+    const onClick = (e) => { e.preventDefault(); onClickRecord = 'clicked'; };
+    const component = mount(
+        <ElectronicResourcesSearch resources={[ { url: 'fakeurl', label: 'fakelabel' } ]} onClick={onClick} bibUrl='fakebiburl'/>
+    )
```

Comment:
This isn't a problem for this `describe`, but consider using `beforeEach` to wrap these two statements. This component only renders content and each `it` is checking only _one_ mounted instance of the component. If it was a more complicated component, the individual `it` blocks could become dependent on one another. But, again, this is not a problem for this case.

Reply:
Good point but in the interest of keeping this moving I'll leave that to another refactor